tests/move/arguments/sources/main.move

```diff
@@ -157,43 +162,44 @@ module transaction_arguments::tx_args_module {
         // deeply_nested_2: vector<Option<vector<vector<vector<Option<u256>>>>>>, // TODO
         // deeply_nested_3: vector<Option<vector<vector<vector<Option<Object<EmptyResource>>>>>>>, // TODO
```

Comment:
It would be good to test vector<vector<value>> especially vector<vector<u8>> and vector<vector<u64>>

Reply:
```rust
entry fun complex_arguments(
    deeply_nested_1: vector<vector<u8>>,
    deeply_nested_2: vector<vector<String>>,
    deeply_nested_3: vector<Option<vector<String>>>,
    deeply_nested_4: vector<vector<Option<vector<String>>>>,
) {
```

Done!